php/meterpreter/meterpreter.php

```diff
@@ -1150,7 +1149,7 @@ function read($resource, $len=null) {
             if ($resource != $msgsock) { my_print("buff: '$buff'"); }
             $r = Array($resource);
         }
-        my_print(sprintf("Done with the big read loop on $resource, got %d bytes", strlen($buff)));
+        my_print(sprintf("Done with the big read loop on $resource, got %d bytes, asked for %d bytes", strlen($buff), $len));
```

Comment:
From line 1139 above, it seems that it could be the case that we "asked for" $md['unread_bytes'] bytes, not necessarily $len...?